### ISSUE TITLE
Ra'ivavae is missing in the data: why?

### ISSUE THREAD

**Issue by LinguList** [lexibank/abvdoceanic#13]
@SimonGreenhill, it seems that [Ra'ivavae](https://abvd.shh.mpg.de/austronesian/language.php?id=1213) is not in the data (id=1213 in ABVD). I wonder how this happened, is it my code, or is it the data that you originally extracted? And is it possible there are more languages missing, or were they excluded for other reasons?

**Comment by LinguList** [comment]
@maryewal, I think that with the other issue in walworthpolynesian, and a check on the data again, this issue can then be solved, so we don't have real flaws, we just had an older version of glottolog, as far as I can see now.

**Comment by SimonGreenhill** [comment]
It's not in the [filtered list](https://github.com/lexibank/abvdoceanic/blob/main/etc/oceanicfiltered.txt) of things to analyse

**Comment by LinguList** [comment]
Ah, thanks! So this is deliberate, right?

**Comment by LinguList** [comment]
Then we can close this issue.

**Comment by SimonGreenhill** [comment]
ja, the phylogenetic analysis is only running off a subset of the 'best' data, not all oceanic in ABVD. I would have thought Ra'ivavae would be in the final list but obviously not (perhaps @marywal can double check)

**Comment by maryewal** [comment]
we did some picking out of Vanuatu and NC, but Ra'ivavae should be in there.

**Comment by SimonGreenhill** [comment]
Hmm, this is Ben's list (i.e. it'll be left out of the phylogenies too). Can we double check this?

**Comment by maryewal** [comment]
Yep, will ask him at today's meeting.

**Comment by LinguList** [comment]
Is Ben in this repository, so we can discuss with him? I think all have 
admin rights, so you can add additional persons to the repository, if 
you want.

**Comment by maryewal** [comment]
We discussed at our CoOL meeting today and Simon is right that it is/was a glottocode issue (where there was the same glottocode for a group of languages, Ben pulled out the one with highest coverage). We are checking that there aren't other such cases like Ra'ivavae/Rurutu for the oceanic tree set, but I'm not sure it matters too much for the sound inventories? 
(ben is @King-Ben)

**Comment by king-ben** [comment]
Ra'ivavae has the same language-level Glottocode as Rurutuan in Glottolog

**Comment by LinguList** [comment]
Yes, @king-ben, one should use the dialect glottocodes instead. The language-dialect distinction is anyway arbitrary in some sense, or does this yield a problem for any further comparisons?
